src/components/pages/frontPage/frontPage.scss

```diff
@@ -7,6 +8,24 @@
   }
 
   &__card-wrapper {
+    padding-right: 10px;
+    padding-left: 10px;
     margin-bottom: 2em;
+    position: relative;
+    width: 100%;
```

Comment:
Style has no effect

Reply:
Not on desktop, but it has on smaller screens